applications/DelaunayMeshingApplication/python_scripts/domain_utilities.py

```diff
@@ -159,7 +159,7 @@ def AddBoundaryNodesToFluidDomains(self,model_part):
 
             print("::[--Domain Utilities-]:: Add boundary nodes to fluid domains ")
 
-            transfer_flags = [KratosMultiphysics.BOUNDARY,KratosMultiphysics.NOT_FLUID]
+            transfer_flags = [KratosMultiphysics.BOUNDARY,KratosMultiphysics.Flags.__unset__(KratosMultiphysics.FLUID)]
```

Comment:
why `__unset__` and not `Unset` in python? Is `__unset__` an speciall method?

Reply:
I don't know :sweat_smile: . The false will be `False()` in Python too.